libvast/builtins/operators/local_remote.cpp

```diff
@@ -83,19 +88,6 @@ class local_remote_operator final : public operator_base {
 template <detail::string_literal Name, operator_location Location>
 class plugin final : public virtual operator_parser_plugin {
 public:
-  auto initialize([[maybe_unused]] const record& plugin_config,
-                  const record& global_config) -> caf::error override {
-    auto allow_unsafe_pipelines
-      = try_get_or(global_config, "tenzir.allow-unsafe-pipelines", false);
-    if (not allow_unsafe_pipelines) {
-      return caf::make_error(
-        ec::invalid_configuration,
-        fmt::format("failed to parse tenzir.allow-unsafe-pipelines option: {}",
-                    allow_unsafe_pipelines.error()));
-    }
-    return {};
-  }
```

Comment:
This did check whether the option was either unset or holds a boolean, which I think is not something that we want to lose.